posthog/hogql_queries/insights/trends/trends_query_builder.py

```diff
@@ -671,7 +671,9 @@ def _events_filter(
     ) -> ast.Expr:
         series = self.series
         filters: list[ast.Expr] = []
-        is_data_warehouse_series = isinstance(series, DataWarehouseNode)
+        is_data_warehouse_event_series = isinstance(series, DataWarehouseNode) and any(
+            series.table_name == modifier.table_name for modifier in self.modifiers.dataWarehouseEventsModifiers
```

Comment:
You'll wanna check `self.modifiers.dataWarehouseEventsModifiers` is not `None`

Reply:
0d95dc2171eb57ee53a5f4b811bd0cd840b3cf5b